ccvm_simulators/solvers/dl_solver.py

```diff
@@ -7,7 +7,7 @@
 import torch.distributions as tdist
 import time
 
-DL_SCALING_MULTIPLIER = 0.5
+DL_SCALING_MULTIPLIER = 0.2
```

Comment:
I have performed some parameter tuning since the first time I set these parameters and have found that for the newly generated random instances these parameters work better. Same goes for the other solvers.

Reply:
That's awesome, thanks.